pythonforandroid/build.py

```diff
@@ -327,46 +374,21 @@ def prepare_build_environment(self,
         if not self.ccache:
             info('ccache is missing, the build will not be optimized in the '
                  'future.')
-        for cython_fn in ("cython", "cython3", "cython2", "cython-2.7"):
-            cython = sh.which(cython_fn)
-            if cython:
-                self.cython = cython
-                break
-        else:
-            raise BuildInterruptingException('No cython binary found.')
-        if not self.cython:
-            ok = False
-            warning("Missing requirement: cython is not installed")
+        self.cython = get_cython_path()
```

Comment:
Not only we moved this block in a `get_cython_path()` helper above, but also removed the `warning()` that seemed dead code since we're raising exception